src/shared/components/mutationTable/column/HgvsgColumnFormatter.tsx

```diff
@@ -16,7 +20,11 @@ export default class HgvsgColumnFormatter {
         if (!hgvsg) {
             return <span />;
         } else {
-            let genomeNexusUrl = buildGenomeNexusHgvsgUrl(hgvsg);
+            let genomeNexusUrl =
```

Comment:
you should put this check inside buildGenomeNexusHgvsgUrl function.   This code shouldn't have to know about 38 vs 37.